src/context.rs

```diff
@@ -84,4 +105,149 @@ impl Context {
       )
     }
   }
+
+  fn get_annex_mut<'a>(
+    &'a self,
+    isolate: &'a mut Isolate,
+  ) -> &'a mut ContextAnnex {
+    assert!(
+      std::ptr::eq(isolate, unsafe { v8__Context__GetIsolate(self) }),
+      "attempted to use Context slots with the wrong Isolate"
+    );
+
+    let num_data_fields =
+      unsafe { v8__Context__GetNumberOfEmbedderDataFields(self) } as c_int;
+    if num_data_fields <= Self::ANNEX_SLOT {
+      let annex = Box::new(ContextAnnex {
+        slots: Default::default(),
+        // To be replaced
```

Comment:
What do you want to replace it with?

Reply:
That comment wasn't a TODO, it was a note that `weak` gets set to a weak handle to the context further down the function. I'll reword it.